src/PlaywrightSharp/Playwright.cs

```diff
@@ -52,26 +44,26 @@ internal Playwright(IChannelOwner parent, string guid, PlaywrightInitializer ini
         /// <inheritdoc/>
         IChannel<Playwright> IChannelOwner<Playwright>.Channel => _channel;
 
-        /// <summary>
-        /// Returns a list of devices to be used with <see cref="IBrowser.NewContextAsync(BrowserContextOptions)"/>.
-        /// </summary>
-        public IReadOnlyDictionary<string, DeviceDescriptor> Devices => _devices;
-
         /// <inheritdoc/>
-        public IChromiumBrowserType Chromium => _initializer.Chromium;
+        public IBrowserType Chromium { get => _initializer.Chromium; set => throw new NotSupportedException(); }
```

Comment:
Can you see if we can assume read-only?

Reply:
Hm, I don't think so, otherwise deserialization will start failing. I'm going to fix it, just not in this PR :)